ament_flake8/ament_flake8/configuration/ament_flake8.ini

```diff
@@ -1,4 +1,6 @@
 [flake8]
-ignore = ''
-max-line-length = 99
+ignore = D100,D101,D102,D103,D104,D203
```

Comment:
`ignore = D100,D101,D102,D103,D104,D105,D203,D212,D404` would better match https://github.com/ament/ament_lint/blob/9eac18aa36dc625c6652c6156e61afd21cd30607/ament_pep257/ament_pep257/main.py#L48 (and below)